### PR TITLE
immudb: init at 1.3.2

### PR DESCRIPTION
**Pull request by dit7ya**
###### Description of changes

immudb is a database with built-in cryptographic proof and verification. It tracks changes in sensitive data and the integrity of the history will be protected by the clients, without the need to trust the database. It can operate both as a key-value store, and/or as relational database (SQL).

https://github.com/codenotary/immudb

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Format this code.

File: pkgs/servers/nosql/immudb/default.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, fetchzip
+, installShellFiles
+}:
+let
```
Comment (AndersonTorres) [inline pkgs/servers/nosql/immudb/default.nix:7]:
```suggestion

let
```

File: pkgs/servers/nosql/immudb/default.nix [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  webconsoleVersion = "1.0.17";
+  webconsoleDist = fetchzip {
+    url = "https://github.com/codenotary/immudb-webconsole/releases/download/v${webconsoleVersion}/immudb-webconsole.tar.gz";
+    sha256 = "sha256-hFSvPwSRXyrSBYktTOwIRa1+aH+mX/scDYDokvZuW1s=";
+  };
+in
```
Comment (AndersonTorres) [inline pkgs/servers/nosql/immudb/default.nix:13]:
```suggestion

in
```

File: pkgs/servers/nosql/immudb/default.nix [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+buildGoModule rec {
+  pname = "immudb";
+  version = "1.3.2";
+  src = fetchFromGitHub {
```
Comment (AndersonTorres) [inline pkgs/servers/nosql/immudb/default.nix:17]:
```suggestion

  src = fetchFromGitHub {
```

File: pkgs/servers/nosql/immudb/default.nix [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  postInstall = ''
+
+      mkdir -p share/completions
+      for executable in immudb immuclient immuadmin; do
+          for shell in bash fish zsh; do
+            $out/bin/$executable completion $shell > share/completions/$executable.$shell
+            installShellCompletion share/completions/$executable.$shell
+          done
+        done
+    '';
```
Comment (AndersonTorres) [inline pkgs/servers/nosql/immudb/default.nix:55]:
```suggestion
  postInstall = ''
    mkdir -p share/completions
    for executable in immudb immuclient immuadmin; do
        for shell in bash fish zsh; do
        $out/bin/$executable completion $shell > share/completions/$executable.$shell
        installShellCompletion share/completions/$executable.$shell
      done
    done
  '';
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Review by viraptor — COMMENTED** [review]
Result of `nixpkgs-review pr 190109` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>immudb</li>
  </ul>
</details>

Apart from the style change requests, it looks ok to me.

File: pkgs/servers/nosql/immudb/default.nix [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+  postInstall = ''
+      mkdir -p share/completions
+      for executable in immudb immuclient immuadmin; do
+          for shell in bash fish zsh; do
+            $out/bin/$executable completion $shell > share/completions/$executable.$shell
+            installShellCompletion share/completions/$executable.$shell
+          done
+        done
+    '';
```
Comment (AndersonTorres) [inline pkgs/servers/nosql/immudb/default.nix:57]: Indent the code:

```suggestion
  postInstall = ''
    mkdir -p share/completions
    for executable in immudb immuclient immuadmin; do
      for shell in bash fish zsh; do
        $out/bin/$executable completion $shell > share/completions/$executable.$shell
        installShellCompletion share/completions/$executable.$shell
      done
    done
  '';
```

File: pkgs/servers/nosql/immudb/default.nix [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+, installShellFiles
+}:
+
+let
+  webconsoleVersion = "1.0.17";
+  webconsoleDist = fetchzip {
+    url = "https://github.com/codenotary/immudb-webconsole/releases/download/v${webconsoleVersion}/immudb-webconsole.tar.gz";
+    sha256 = "sha256-hFSvPwSRXyrSBYktTOwIRa1+aH+mX/scDYDokvZuW1s=";
+  };
+
```
Comment (AndersonTorres) [inline pkgs/servers/nosql/immudb/default.nix:14]:
```suggestion
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
And this ends the cosmetical part. Otherwise LGTM.

**Review by AndersonTorres — APPROVED** [review]
LGTM

**Pull request merged**